47 frontend merge changes of styles from user profile branch to the updated main branch

Jialipeng8 opened the pull request:
style changes for details see pictures attached 
![Screenshot 2024-05-18 at 4 22 11 PM](https://github.com/imran072/request-forum-app/assets/126379251/785194a2-ad6b-4593-815e-1775a9938687)
![Screenshot 2024-05-18 at 4 20 49 PM](https://github.com/imran072/request-forum-app/assets/126379251/4ee6730b-7071-48d7-95ca-492cd859210b)
![Screenshot 2024-05-18 at 4 20 43 PM](https://github.com/imran072/request-forum-app/assets/126379251/388db3c1-6215-4d34-99df-f0776d7a48f4)

Scarlettzpl committed:
line 11 default.html removed class='d-flex flex-column min-vh-100'

Scarlettzpl committed:
line 11 default.html removed class='d-flex flex-column min-vh-100'

Scarlettzpl committed:
made the style of signup and login pages consistent, customised the style of logout button to make it consistent in the navbar

Scarlettzpl committed:
modified the class for login_form in line8 and removed the inline css in line 8,13,17,22,33

Scarlettzpl committed:
line486 maincss file added add listing class make the background of the pages consistent

Scarlettzpl committed:
modified and added div and classes to make it consistent with login and signup html

Scarlettzpl committed:
line 11 changed body class name

Scarlettzpl committed:
reversed the sell my ev in navbar.html

Scarlettzpl committed:
modify the class name for style changes

Scarlettzpl committed:
changed line8 class name for modifying the style in boostrap

Scarlettzpl committed:
modified line 6, 12, 35, 61 class name for modifying styles

zhanerken commented:
@Scarlettzpl @Jialipeng8 there are conflicts that has to be resolved, besides that there multiple files that are altered and I'm not sure if it is aligned well with work of @pbuslee.

Jialipeng8 commented:
hey please see the attched, the only changes that conflicts it's only i changed the "MY profile" to "profile" and changed the style that doesn't significantly change anything.
![Screenshot 2024-05-18 at 7 11 01 PM](https://github.com/imran072/request-forum-app/assets/126379251/9f4203de-bbb8-472c-ac91-b9aaf7ce12f7)

imran072 commented:
Closing as it was late to check the compatibility of the new style with all the existing flows and pages.
